Description: Fix 32-bit build with 64-bit time_t
Author: Andreas Beckmann <anbe@debian.org>
Bug-Debian: https://bugs.debian.org/1081646

--- a/xdr.c
+++ b/xdr.c
@@ -61,8 +61,13 @@ xdr_msg_update(XDR *xdrs, struct msg_upd
 static int
 xdr_msg_delayinfo(XDR *xdrs, struct msg_delayinfo *delayinfo)
 {
+#if _TIME_BITS == 64
+	X(xdr_int64_t(xdrs, &delayinfo->delaytv.tv_sec));
+	X(xdr_int64_t(xdrs, &delayinfo->delaytv.tv_usec));
+#else
 	X(xdr_long(xdrs, &delayinfo->delaytv.tv_sec));
 	X(xdr_long(xdrs, &delayinfo->delaytv.tv_usec));
+#endif
 	X(xdr_int(xdrs, (int *)&delayinfo->len));
 
 	return (TRUE);
