Description: Changes to configure.in as of trickle 1.07-10.1

--- trickle-1.07.orig/configure.in
+++ trickle-1.07/configure.in
@@ -67,8 +67,9 @@
 AC_CHECK_TYPE(u_int32_t, unsigned int)
 AC_CHECK_TYPE(u_int16_t, unsigned short)
 AC_CHECK_TYPE(u_int8_t, unsigned char)
-AC_CHECK_TYPE(in_addr_t, unsigned long)
-
+AC_CHECK_TYPE(in_addr_t, , [AC_DEFINE(in_addr_t, unsigned long)], [AC_INCLUDES_DEFAULT
+#include <netinet/in.h>
+])
 dnl Checks for library functions.
 AC_PROG_GCC_TRADITIONAL
 AC_TYPE_SIGNAL
