Description: Make test target build under dh compat level 13
Author: Axel Beckert <abe@debian.org>

--- a/test.c
+++ b/test.c
@@ -4,7 +4,7 @@
 #include "config.h"
 #endif /* HAVE_CONFIG_H */
 
-#include <sys/tree.h>
+#include <bsd/sys/tree.h>
 #include <sys/queue.h>
 #include <sys/socket.h>
 #include <sys/un.h>
@@ -18,3 +18,6 @@
 #include <netinet/in.h>
 #include <netinet/in_systm.h>
 
+void main () {
+  return;
+}
